sdk-setup/src/sdk-manage

```diff
@@ -165,6 +165,12 @@ download() {
 
     url=${url#file://}
 
+    if [[ $url =~ \.7z$ ]]; then
+        local_file_hint="$local_file_hint.7z"
+    else
+        local_file_hint="$local_file_hint.bz2"
+    fi
+
```

Comment:
I would consider using `local_file_hint+=.tar.${url##*.}` and omitting ".tar" in the arguments when calling this function, i.e., passing just the base name without extension.